packages/react-native-bottom-tabs/ios/TabView/NewTabView.swift

```diff
@@ -51,5 +52,75 @@ struct NewTabView: AnyTabView {
     .measureView { size in
       onLayout(size)
     }
+    .modifier(ConditionalBottomAccessoryModifier(props: props))
+  }
+}
+
+struct ConditionalBottomAccessoryModifier: ViewModifier {
+  let props: TabViewProps
+  
+  // Check if there's a bottom accessory component view
+  private var hasBottomAccessory: Bool {
+    props.children.contains { child in
+      let className = String(describing: type(of: child.view))
+      return className == "RCTBottomAccessoryComponentView"
+    }
+  }
+  
+  // Find the bottom accessory view
+  private var bottomAccessoryView: PlatformView? {
+    props.children.first { child in
+      let className = String(describing: type(of: child.view))
+      return className == "RCTBottomAccessoryComponentView"
+    }?.view
+  }
+  
+  func body(content: Content) -> some View {
+    if #available(iOS 26.0, macOS 26.0, tvOS 26.0, visionOS 3.0, *), hasBottomAccessory {
+        content
+        .tabViewBottomAccessory {
+          renderBottomAccessoryView()
+        }
+    } else {
+      content
+    }
+  }
+  
+  @ViewBuilder
+  private func renderBottomAccessoryView() -> some View {
+    if let accessoryView = bottomAccessoryView {
+      if #available(iOS 26.0, *) {
+        BottomAccessoryRepresentableView(view: accessoryView)
+      }
+    }
+  }
+}
+
+@available(iOS 26.0, *)
+struct BottomAccessoryRepresentableView: PlatformViewRepresentable {
+  @Environment(\.tabViewBottomAccessoryPlacement) var tabViewBottomAccessoryPlacement
+  var view: PlatformView
+
+  func makeUIView(context: Context) -> PlatformView {
+    emitPlacementChanged(for: view)
+    return view
+  }
+
+  func updateUIView(_ uiView: PlatformView, context: Context) {
+    emitPlacementChanged(for: view)
+  }
+  
+  private func emitPlacementChanged(for uiView: PlatformView) {
+    let selectorString = "emitOnPlacementChanged:"
+    let selector = NSSelectorFromString(selectorString)
```

Comment:
I think this could be solved by a shared protocol between `RCTBottomAccessoryView` and `NewTabView` this way we would have type safety

Reply:
Sure! I will do that asap